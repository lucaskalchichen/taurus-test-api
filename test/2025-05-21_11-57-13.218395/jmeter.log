2025-05-21 11:57:16,904 INFO o.a.j.u.JMeterUtils: Setting Locale to en
2025-05-21 11:57:16,914 INFO o.a.j.JMeter: Loading user properties from: /home/lucas/.bzt/jmeter-taurus/5.5/bin/user.properties
2025-05-21 11:57:16,915 INFO o.a.j.JMeter: Loading system properties from: /home/lucas/.bzt/jmeter-taurus/5.5/bin/system.properties
2025-05-21 11:57:16,915 INFO o.a.j.JMeter: Loading additional properties from: /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-57-13.218395/jmeter-bzt.properties
2025-05-21 11:57:16,917 INFO o.a.j.JMeter: Setting System properties from file: /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-57-13.218395/system.properties
2025-05-21 11:57:16,920 INFO o.a.j.JMeter: Copyright (c) 1998-2022 The Apache Software Foundation
2025-05-21 11:57:16,921 INFO o.a.j.JMeter: Version 5.5
2025-05-21 11:57:16,921 INFO o.a.j.JMeter: java.version=11.0.27
2025-05-21 11:57:16,921 INFO o.a.j.JMeter: java.vm.name=OpenJDK 64-Bit Server VM
2025-05-21 11:57:16,921 INFO o.a.j.JMeter: os.name=Linux
2025-05-21 11:57:16,921 INFO o.a.j.JMeter: os.arch=amd64
2025-05-21 11:57:16,921 INFO o.a.j.JMeter: os.version=5.15.167.4-microsoft-standard-WSL2
2025-05-21 11:57:16,921 INFO o.a.j.JMeter: file.encoding=UTF-8
2025-05-21 11:57:16,922 INFO o.a.j.JMeter: java.awt.headless=true
2025-05-21 11:57:16,922 INFO o.a.j.JMeter: Max memory     =4160749568
2025-05-21 11:57:16,922 INFO o.a.j.JMeter: Available Processors =28
2025-05-21 11:57:16,925 INFO o.a.j.JMeter: Default Locale=English
2025-05-21 11:57:16,925 INFO o.a.j.JMeter: JMeter  Locale=English
2025-05-21 11:57:16,925 INFO o.a.j.JMeter: JMeterHome=/home/lucas/.bzt/jmeter-taurus/5.5
2025-05-21 11:57:16,925 INFO o.a.j.JMeter: user.dir  =/mnt/d/Universidad/recursos/taurus api/test
2025-05-21 11:57:16,925 INFO o.a.j.JMeter: PWD       =/mnt/d/Universidad/recursos/taurus api/test
2025-05-21 11:57:16,931 INFO o.a.j.JMeter: IP: 127.0.1.1 Name: kalchi-pc FullName: kalchi-pc.
2025-05-21 11:57:16,931 INFO o.a.j.JMeter: user.classpath=/mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-57-13.218395:/mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-57-13.218395
2025-05-21 11:57:16,931 INFO o.a.j.JMeter: Adding to classpath and loader: /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-57-13.218395
2025-05-21 11:57:16,933 INFO o.a.j.JMeter: Adding to classpath and loader: /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-57-13.218395
2025-05-21 11:57:16,938 INFO o.a.j.s.FileServer: Default base='/mnt/d/Universidad/recursos/taurus api/test'
2025-05-21 11:57:16,938 INFO o.a.j.s.FileServer: Set new base='/mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-57-13.218395'
2025-05-21 11:57:17,015 INFO o.a.j.s.SaveService: Testplan (JMX) version: 2.2. Testlog (JTL) version: 2.2
2025-05-21 11:57:17,026 INFO o.a.j.s.SaveService: Using SaveService properties version 5.0
2025-05-21 11:57:17,028 INFO o.a.j.s.SaveService: Using SaveService properties file encoding UTF-8
2025-05-21 11:57:17,030 INFO o.a.j.s.SaveService: Loading file: /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-57-13.218395/modified_requests.jmx
2025-05-21 11:57:17,047 INFO o.a.j.p.h.c.CacheManager: Will only cache the following methods: [GET]
2025-05-21 11:57:17,049 INFO o.a.j.p.h.c.CookieManager: Settings: Delete null: true Check: true Allow variable: true Save: false Prefix: COOKIE_
2025-05-21 11:57:17,057 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/html is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2025-05-21 11:57:17,058 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for application/xhtml+xml is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2025-05-21 11:57:17,058 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for application/xml is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2025-05-21 11:57:17,058 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/xml is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2025-05-21 11:57:17,058 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/vnd.wap.wml is org.apache.jmeter.protocol.http.parser.RegexpHTMLParser
2025-05-21 11:57:17,058 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/css is org.apache.jmeter.protocol.http.parser.CssParser
2025-05-21 11:57:17,077 INFO o.a.j.e.StandardJMeterEngine: Running the test!
2025-05-21 11:57:17,077 INFO o.a.j.s.SampleEvent: List of sample_variables: []
2025-05-21 11:57:17,077 INFO o.a.j.s.SampleEvent: List of sample_variables: []
2025-05-21 11:57:17,085 INFO o.a.j.e.u.CompoundVariable: Note: Function class names must contain the string: '.functions.'
2025-05-21 11:57:17,086 INFO o.a.j.e.u.CompoundVariable: Note: Function class names must not contain the string: '.gui.'
2025-05-21 11:57:17,222 INFO o.a.j.JMeter: Running test (1747839437222)
2025-05-21 11:57:17,230 INFO o.a.j.e.StandardJMeterEngine: Starting ThreadGroup: 1 : api-test
2025-05-21 11:57:17,231 INFO o.a.j.e.StandardJMeterEngine: Starting 42 threads for group api-test.
2025-05-21 11:57:17,231 INFO o.a.j.e.StandardJMeterEngine: Thread will continue on error
2025-05-21 11:57:17,231 INFO o.a.j.t.ThreadGroup: Starting thread group... number=1 threads=42 ramp-up=3 delayedStart=false
2025-05-21 11:57:17,237 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-1
2025-05-21 11:57:17,250 INFO o.a.j.p.h.s.HTTPHCAbstractImpl: Local host = kalchi-pc
2025-05-21 11:57:17,251 INFO o.a.j.t.ThreadGroup: Started thread group number 1
2025-05-21 11:57:17,251 INFO o.a.j.e.StandardJMeterEngine: All thread groups have been started
2025-05-21 11:57:17,253 INFO o.a.j.p.h.s.HTTPHC4Impl: HTTP request retry count = 0
2025-05-21 11:57:17,253 INFO o.a.j.s.SampleResult: Note: Sample TimeStamps are START times
2025-05-21 11:57:17,253 INFO o.a.j.s.SampleResult: sampleresult.default.encoding is set to UTF-8
2025-05-21 11:57:17,254 INFO o.a.j.s.SampleResult: sampleresult.useNanoTime=true
2025-05-21 11:57:17,254 INFO o.a.j.s.SampleResult: sampleresult.nanoThreadSleep=5000
2025-05-21 11:57:17,306 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-2
2025-05-21 11:57:17,375 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-3
2025-05-21 11:57:17,445 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-4
2025-05-21 11:57:17,518 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-5
2025-05-21 11:57:17,589 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-6
2025-05-21 11:57:17,659 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-7
2025-05-21 11:57:17,729 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-8
2025-05-21 11:57:17,766 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-1
2025-05-21 11:57:17,766 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-1
2025-05-21 11:57:17,800 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-9
2025-05-21 11:57:17,871 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-10
2025-05-21 11:57:17,942 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-11
2025-05-21 11:57:18,012 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-12
2025-05-21 11:57:18,084 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-13
2025-05-21 11:57:18,156 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-14
2025-05-21 11:57:18,226 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-15
2025-05-21 11:57:18,246 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-2
2025-05-21 11:57:18,246 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-2
2025-05-21 11:57:18,296 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-16
2025-05-21 11:57:18,367 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-17
2025-05-21 11:57:18,439 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-18
2025-05-21 11:57:18,511 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-19
2025-05-21 11:57:18,580 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-20
2025-05-21 11:57:18,652 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-21
2025-05-21 11:57:18,706 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-3
2025-05-21 11:57:18,707 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-3
2025-05-21 11:57:18,730 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-22
2025-05-21 11:57:18,793 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-23
2025-05-21 11:57:18,864 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-24
2025-05-21 11:57:18,936 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-25
2025-05-21 11:57:19,007 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-26
2025-05-21 11:57:19,078 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-27
2025-05-21 11:57:19,148 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-28
2025-05-21 11:57:19,172 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-4
2025-05-21 11:57:19,173 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-4
2025-05-21 11:57:19,220 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-29
2025-05-21 11:57:19,291 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-30
2025-05-21 11:57:19,362 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-31
2025-05-21 11:57:19,433 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-32
2025-05-21 11:57:19,503 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-33
2025-05-21 11:57:19,575 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-34
2025-05-21 11:57:19,625 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-5
2025-05-21 11:57:19,625 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-5
2025-05-21 11:57:19,647 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-35
2025-05-21 11:57:19,719 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-36
2025-05-21 11:57:19,787 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-37
2025-05-21 11:57:19,858 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-38
2025-05-21 11:57:19,931 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-39
2025-05-21 11:57:20,001 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-40
2025-05-21 11:57:20,072 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-41
2025-05-21 11:57:20,092 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-6
2025-05-21 11:57:20,093 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-6
2025-05-21 11:57:20,142 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-42
2025-05-21 11:57:20,546 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-7
2025-05-21 11:57:20,546 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-7
2025-05-21 11:57:21,006 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-8
2025-05-21 11:57:21,007 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-8
2025-05-21 11:57:21,449 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-9
2025-05-21 11:57:21,449 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-9
2025-05-21 11:57:21,910 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-10
2025-05-21 11:57:21,910 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-10
2025-05-21 11:57:22,414 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-12
2025-05-21 11:57:22,415 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-12
2025-05-21 11:57:22,848 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-11
2025-05-21 11:57:22,848 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-11
2025-05-21 11:57:23,159 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-13
2025-05-21 11:57:23,159 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-13
2025-05-21 11:57:23,622 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-14
2025-05-21 11:57:23,622 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-14
2025-05-21 11:57:24,084 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-15
2025-05-21 11:57:24,084 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-15
2025-05-21 11:57:24,518 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-16
2025-05-21 11:57:24,518 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-16
2025-05-21 11:57:24,958 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-17
2025-05-21 11:57:24,959 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-17
2025-05-21 11:57:25,410 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-18
2025-05-21 11:57:25,410 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-18
2025-05-21 11:57:25,911 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-20
2025-05-21 11:57:25,911 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-20
2025-05-21 11:57:26,303 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-19
2025-05-21 11:57:26,303 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-19
2025-05-21 11:57:26,718 INFO o.a.j.JMeter: Command: Shutdown received from /127.0.0.1
2025-05-21 11:57:26,719 INFO o.a.j.t.JMeterThread: Stopping: api-test 1-25
2025-05-21 11:57:26,719 INFO o.a.j.t.JMeterThread: Stopping: api-test 1-40
2025-05-21 11:57:26,719 INFO o.a.j.t.JMeterThread: Stopping: api-test 1-29
2025-05-21 11:57:26,719 INFO o.a.j.t.JMeterThread: Stopping: api-test 1-33
2025-05-21 11:57:26,719 INFO o.a.j.t.JMeterThread: Stopping: api-test 1-23
2025-05-21 11:57:26,720 INFO o.a.j.t.JMeterThread: Stopping: api-test 1-31
2025-05-21 11:57:26,720 INFO o.a.j.t.JMeterThread: Stopping: api-test 1-35
2025-05-21 11:57:26,720 INFO o.a.j.t.JMeterThread: Stopping: api-test 1-38
2025-05-21 11:57:26,720 INFO o.a.j.t.JMeterThread: Stopping: api-test 1-32
2025-05-21 11:57:26,720 INFO o.a.j.t.JMeterThread: Stopping: api-test 1-37
2025-05-21 11:57:26,720 INFO o.a.j.t.JMeterThread: Stopping: api-test 1-34
2025-05-21 11:57:26,720 INFO o.a.j.t.JMeterThread: Stopping: api-test 1-41
2025-05-21 11:57:26,720 INFO o.a.j.t.JMeterThread: Stopping: api-test 1-39
2025-05-21 11:57:26,720 INFO o.a.j.t.JMeterThread: Stopping: api-test 1-21
2025-05-21 11:57:26,720 INFO o.a.j.t.JMeterThread: Stopping: api-test 1-22
2025-05-21 11:57:26,721 INFO o.a.j.t.JMeterThread: Stopping: api-test 1-24
2025-05-21 11:57:26,721 INFO o.a.j.t.JMeterThread: Stopping: api-test 1-26
2025-05-21 11:57:26,721 INFO o.a.j.t.JMeterThread: Stopping: api-test 1-28
2025-05-21 11:57:26,721 INFO o.a.j.t.JMeterThread: Stopping: api-test 1-36
2025-05-21 11:57:26,721 INFO o.a.j.t.JMeterThread: Stopping: api-test 1-30
2025-05-21 11:57:26,721 INFO o.a.j.t.JMeterThread: Stopping: api-test 1-27
2025-05-21 11:57:26,721 INFO o.a.j.t.JMeterThread: Stopping: api-test 1-42
2025-05-21 11:57:26,743 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-21
2025-05-21 11:57:26,743 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-21
2025-05-21 11:57:26,779 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-22
2025-05-21 11:57:26,797 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-24
2025-05-21 11:57:26,839 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-23
2025-05-21 11:57:26,847 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-25
2025-05-21 11:57:26,856 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-26
2025-05-21 11:57:26,865 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-27
2025-05-21 11:57:26,874 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-28
2025-05-21 11:57:26,883 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-29
2025-05-21 11:57:26,892 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-30
2025-05-21 11:57:26,901 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-31
2025-05-21 11:57:26,910 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-32
2025-05-21 11:57:26,919 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-33
2025-05-21 11:57:26,928 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-34
2025-05-21 11:57:26,937 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-35
2025-05-21 11:57:26,946 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-36
2025-05-21 11:57:26,955 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-37
2025-05-21 11:57:26,963 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-38
2025-05-21 11:57:26,972 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-39
2025-05-21 11:57:26,982 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-40
2025-05-21 11:57:26,991 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-41
2025-05-21 11:57:26,999 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-42
2025-05-21 11:57:27,000 INFO o.a.j.e.StandardJMeterEngine: Notifying test listeners of end of test
2025-05-21 11:57:27,001 INFO o.a.j.e.StandardJMeterEngine: Forced JVM shutdown requested at end of test
