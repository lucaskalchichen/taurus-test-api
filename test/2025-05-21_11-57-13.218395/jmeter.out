Created the tree successfully using /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-57-13.218395/modified_requests.jmx
Starting standalone test @ 2025 May 21 11:57:17 ART (1747839437072)
Waiting for possible Shutdown/StopTestNow/HeapDump/ThreadDump message on port 4445
Command: Shutdown received from /127.0.0.1
Tidying up ...    @ 2025 May 21 11:57:27 ART (1747839447001)
... end of run
